api/types/session_tracker.go

```diff
@@ -39,6 +39,7 @@ const (
 	DatabaseSessionKind       SessionKind = "db"
 	AppSessionKind            SessionKind = "app"
 	WindowsDesktopSessionKind SessionKind = "desktop"
+	UnknownSessionKind        SessionKind = ""
```

Comment:
Could you add a comment explaining the purpose of UnknownSessionKind and describing how it might occur?